cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -2491,6 +2496,21 @@ impl MachInst for Inst {
         }
     }
 
+    fn is_included_in_clobbers(&self) -> bool {
+        // We exclude call instructions from the clobber-set when they are calls
+        // from caller to callee with the same ABI. Such calls cannot possibly
+        // force any new registers to be saved in the prologue, because anything
+        // that the callee clobbers, the caller is also allowed to clobber. This
+        // both saves work and enables us to more precisely follow the
+        // half-caller-save, half-callee-save SysV ABI for some vector
+        // registers.
```

Comment:
Maybe worth adding a back reference to the other comment (that references this one) around callee-saved in call sites?

Reply:
Done!